includes/class-mailer.php

```diff
@@ -126,6 +126,11 @@ public static function comment_notification_text( $message, $comment_id ) {
 	 * @param int   $user_id  The id of the local blog-user.
 	 */
 	public static function new_follower( $activity, $user_id ) {
+		// Do not send notifications to the Application user.
+		if ( Actors::APPLICATION_USER_ID === $user_id ) {
```

Comment:
With Application users not being able to be followed, when would this condition be true?